_posts/en/newsletters/2020-03-18-newsletter.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Bitcoin Optech Newsletter #89'
+permalink: /en/newsletters/2020/03/18/
+name: 2020-03-18-newsletter
+slug: 2020-03-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes an update to a proposed standard for
+LN and includes our regular sections about notable changes to services,
+client software, and popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed watchtower BOLT updated:** Sergi Delgado Segura
+  [emailed][segura email] the Lightning-Dev mailing list an [updated
+  version][watchtower bolt] of a suggested protocol for
+  [watchtower][topic watchtowers] communication.  See [Newsletter
+  #75][news75 watchtower] for our original description of this proposal.  According to
+  Segura, the update includes details about "user accounts, payment
+  methods, and message signing."  His email also provides a list of
+  features he would like to add, with discussion about each near the end
+  of the email.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Coinbase withdrawal transactions now using batching:** Coinbase has rolled
+out [batch withdrawals][coinbase batching blog] that they estimate will reduce
+their load on the Bitcoin network by 50%. Instead of each withdrawal payment
+generating a single onchain transaction, multiple payments will be
+[combined into a single transaction][scaling payment batching] once every 10 minutes.
+
+- **Bitstamp supports bech32:** Bitstamp users can now benefit from using native
```

Comment:
Perhaps hoist "deposits and withdrawals" to the bitstamp title for clear quick comparison to the derebit title while skim reading.